hawkular-alerters/pom.xml

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright 2015-2017 Red Hat, Inc. and/or its affiliates
+    and other contributors as indicated by the @author tags.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.hawkular.alerts</groupId>
+    <artifactId>hawkular-alerts</artifactId>
+    <version>1.6.0.Final-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>hawkular-alerters</artifactId>
+  <packaging>pom</packaging>
+
+  <name>Hawkular Alerting: Alerters parent</name>
+  <description>Parent module of alerter artifacts</description>
+
+  <profiles>
+    <profile>
+      <id>default</id>
+      <modules>
+        <module>hawkular-elasticsearch-alerter</module>
+      </modules>
+      <activation>
+        <activeByDefault>true</activeByDefault>
+      </activation>
+    </profile>
+
+    <profile>
+      <id>openshift</id>
+      <modules>
+        <module>hawkular-elasticsearch-alerter</module>
+      </modules>
+    </profile>
+  </profiles>
+
+</project>
```

Comment:
I've been trying to decide whether this should really be part of hAlerting or whether it should be a separate project. It looks to be a generic extension for interacting with an ES db, and that is cool, but it makes me wonder it should be versioned with alerting or on its own schedule.  I suppose this way is easiest, because it interacts with the alerting Java API, it should be deployed against the same version of hAlerting.  Just thinking out loud, I think I am OK with keeping it in the hAlerts umbrella.